File: python/7 kyu/344.py
# https://www.codewars.com/kata/590ac6b9be4dff49b0000042/train/python

# Passed

def convert(degrees): 
    degrees_val = int(degrees)
    minutes = (degrees % 1)
    
    minutes_val = int(minutes * 60)
    seconds_val = round(((minutes * 60) % 1) * 60)
    if seconds_val == 60:
        minutes_val += 1
        seconds_val = 0
        
    if minutes_val == 60:
        degrees_val += 1
        minutes_val = 0
        
    result = [degrees_val]
    if minutes_val:
        result.append(minutes_val)
        
        if seconds_val:
            result.append(seconds_val)
    else:
        if seconds_val:
            result.extend([minutes_val, seconds_val])

    return result

output = convert(40.567)
print(output)

File: nodejs/8 kyu/344.js
// https://www.codewars.com/kata/57eae65a4321032ce000002d/train/javascript

// Passed

function fakeBin(x) {
  let result = x
    .split("")
    .map((digit) => (Number(digit) < 5 ? "0" : "1"))
    .join("");
  return result;
}

const output = fakeBin("509321967506747");
console.log(output);